br/pkg/stream/rewrite_meta_rawkv.go

```diff
@@ -260,22 +145,10 @@ func (sr *SchemasReplace) rewriteDBInfo(value []byte) ([]byte, error) {
 		return nil, errors.Trace(err)
 	}
 
-	if sr.IsPreConsturctMapStatus() {
-		if dr, exist := sr.DbMap[dbInfo.ID]; !exist {
-			newID, err := sr.genGenGlobalID(context.Background())
-			if err != nil {
-				return nil, errors.Trace(err)
-			}
-			sr.DbMap[dbInfo.ID] = NewDBReplace(dbInfo.Name.O, newID)
-		} else {
-			dr.Name = dbInfo.Name.O
-		}
-		return nil, nil
-	}
-
 	dbMap, exist := sr.DbMap[dbInfo.ID]
 	if !exist {
-		return nil, errors.Annotatef(berrors.ErrInvalidArgument, "failed to find id:%v in maps", dbInfo.ID)
+		// db filtered out
+		return nil, nil
```

Comment:
here should only filtered out table/db will not be able to find the mapping. Right now filtered out tables are only some system table. 
Might need some safer mechanism to check if indeed they are filtered out tables, not something we missed due to error in the building id map phase .